src/CoolPropLib.cpp

```diff
@@ -131,6 +131,10 @@ EXPORT_CODE int CONVENTION set_reference_stateD(const char *Ref, double T, doubl
 EXPORT_CODE double CONVENTION Props1(const char *FluidName, const char *Output){
     fpu_reset_guard guard;
```

Comment:
Ian, could you comment on this line:
I'm unsure if fpu reset is required here.
In this function, Props1SI already does that; but after that, two float-point operations happen:

1. ValidNumber() compares (potentially _HUGE) to max/min float (I don't know if this may set some fpu state flag);
2. Valid result is divided by 1000.0. I suppose that underflow is possible here.

So, is the fpu reset actually needed here?